packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-general-information-tile/OrganizationGeneralInformationTile.component.tsx

```diff
@@ -39,72 +33,86 @@ export default function OrganizationGenerationInformationTile({
-          <OsdsIcon
-            aria-label="edit"
-            className="mx-6 cursor-pointer"
-            name={ODS_ICON_NAME.PEN}
-            size={ODS_ICON_SIZE.xxs}
-            color={ODS_THEME_COLOR_INTENT.primary}
-            onClick={() => navigate(subRoutes.editName)}
-          />
-        </div>
-        <OsdsDivider separator />
-        <div className="flex flex-col mb-3">
-          <TileSubtitle>{t('managed_vcd_dashboard_description')}</TileSubtitle>
-          <div className="flex flex-row justify-between items-center">
-            <Description>
-              {vcdOrganization?.currentState?.description}
-            </Description>
-            <OsdsIcon
-              aria-label="edit"
-              className="mx-6 cursor-pointer"
-              name={ODS_ICON_NAME.PEN}
-              size={ODS_ICON_SIZE.xxs}
-              color={ODS_THEME_COLOR_INTENT.primary}
-              onClick={() => navigate(subRoutes.editDescription)}
+    <DashboardTile
+      title={t('managed_vcd_dashboard_general_information')}
+      items={[
+        {
+          id: 'name',
+          label: t('managed_vcd_dashboard_name'),
+          value: (
+            <div className="flex justify-between items-center">
+              <Description>
+                {vcdOrganization?.currentState?.fullName}
+              </Description>
+              <OsdsIcon
+                aria-label="edit"
+                className="mx-6 cursor-pointer"
+                name={ODS_ICON_NAME.PEN}
+                size={ODS_ICON_SIZE.xxs}
+                color={ODS_THEME_COLOR_INTENT.primary}
+                onClick={() => navigate(subRoutes.editName)}
+              />
+            </div>
+          ),
+        },
+        {
+          id: 'description',
+          label: t('managed_vcd_dashboard_description'),
+          value: (
+            <div className="flex justify-between items-center">
+              <Description>
+                {vcdOrganization?.currentState?.description}
+              </Description>
+              <OsdsIcon
+                aria-label="edit"
+                className="mx-6 cursor-pointer"
+                name={ODS_ICON_NAME.PEN}
+                size={ODS_ICON_SIZE.xxs}
+                color={ODS_THEME_COLOR_INTENT.primary}
+                onClick={() => navigate(subRoutes.editDescription)}
+              />
+            </div>
+          ),
+        },
+        {
+          id: 'location',
+          label: t('managed_vcd_dashboard_localisation'),
+          value: (
+            <OsdsText
+              className="mb-4"
+              level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+              color={ODS_THEME_COLOR_INTENT.text}
+              hue={ODS_THEME_COLOR_HUE._500}
+            >
+              <RegionLabel
+                code={vcdOrganization?.currentState?.region}
+              ></RegionLabel>
```

Comment:
**nitpick:** We have a new component on `manager-components` from https://github.com/ovh/manager/issues/12226. Can you please plan to remove this component at the µ-app?